frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -953,12 +959,20 @@ function ContextMenuContainer(props: Props) {
           // @ts-ignore
           ref={inputRef}
         />
-        <ContextMenuInner {...props} inputRef={inputRef} />
+        <ContextMenuContext.Provider value={inputRef}>{props.children}</ContextMenuContext.Provider>
       </div>
     </React.Fragment>
   );
 }
 
+function ContextMenuContainer(props: Props) {
+  return (
+    <GenericContextMenuContainer {...props}>
+      <ContextMenuInner {...props} />
+    </GenericContextMenuContainer>
+  );
+}
+
```

Comment:
It looks to me like these changes introduced some kind of bug I still cannot understand. Due to some reason, the hooks of ContextMenuInner are not called consistently in the same order.
<img width="1424" alt="image" src="https://user-images.githubusercontent.com/39529669/216655031-4bf21223-1db7-48a0-909e-0376b29ba162.png">
I'll open an issue for this.